cosmos/cleanup.py

```diff
@@ -0,0 +1,95 @@
+"""
+Use this script locally to identify broken symbolic links or recursive loops locally:
+  $ python -m cosmos.cleanup -p <dir-path>
+
+To delete the issues identified, run:
+  $ python -m cosmos.cleanup -p <dir-path> -d
+"""
+
+import argparse
+import logging
+import os
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+
+
+def identify_broken_symbolic_links(dir_path: str, should_delete: bool = False) -> None:
+    """
+    Given a directory, recursively inspect it in search for symbolic links.
+    If should_delete is set to True, delete the symbolic links identified.
+
+    :param dir_path: Path to the directory to be analysed
+    :param should_delete: Users should set to True if they want the method to not only identify but also delete these links.
+    """
+    logger.info(f"Inspecting the directory {dir_path} for broken symbolic links.")
+    filepaths = []
+    broken_symlinks_count = 0
+    deleted_symlinks_count = 0
+    for root_dir, dirs, files in os.walk(dir_path):
+        paths = [os.path.join(root_dir, filepath) for filepath in files]
+        filepaths.extend(paths)
+
+    for filepath in filepaths:
+        try:
+            os.stat(filepath)
+        except OSError:
+            broken_symlinks_count += 1
+            logger.warning(f"The folder {dir_path} contains a symbolic link to a non-existent file: {filepath}")
+            if should_delete:
+                logger.info(f"Deleting the invalid symbolic link: {filepath}")
+                os.unlink(filepath)
+                deleted_symlinks_count += 1
+
+    logger.info(
+        f"After inspecting {dir_path}, identified {broken_symlinks_count} broken links and deleted {deleted_symlinks_count} of them."
+    )
+
+
+# Airflow DAG parsing fails if recursive loops are found, so this method cannot be used from within an Airflow task
```

Comment:
Should we also add an example here on how to call this and where to call this like mentioned in the PR description.

Or should we also create a public docs page listing the steps that we can share with users?

Reply:
I think a short docs will be great and also we could render example in the docs